lib/execution_engine2/sdk/SDKMethodRunner.py

```diff
@@ -158,6 +163,12 @@ def get_catalog(self) -> Catalog:
         """
         return self.catalog
 
+    def get_catalog_cache(self) -> CatalogCache:
```

Comment:
Add a line in the SDKMR getters test for this method here: https://github.com/kbase/execution_engine2/pull/391/files?file-filters%5B%5D=.py&file-filters%5B%5D=.txt#diff-d7d713ca6587e357238406d58fce2bf6e82ddb5339eb51dfafeaf6699103e3bfR171

Reply:
done

Reply:
At first I couldn't figure out what that test was trying to do and then I realized the cache isn't mocked, it's created in the SDKMR init

anyway :+1: